tests/tox.sh

```diff
@@ -62,15 +62,9 @@ if [ "${containers_to_remove}" ]; then
     sudo docker rm -f $@ ${containers_to_remove} || echo failed to remove containers
 fi
 
-# copy the files to the root for the
-# types of images we're going to build
-mkdir -p {base,daemon,demo}
-# starting with kraken, the base image does not exist
-cp -Lrv ceph-releases/$CEPH_STABLE_RELEASE/$IMAGE_DISTRO/base/* base || true
-cp -Lrv ceph-releases/$CEPH_STABLE_RELEASE/$IMAGE_DISTRO/daemon/* daemon
-cp -Lrv ceph-releases/$CEPH_STABLE_RELEASE/$IMAGE_DISTRO/demo/* demo || true
-
-bash "$WORKSPACE"/travis-builds/build_imgs.sh
+cd "$WORKSPACE"
+FLAVORS_TO_BUILD=( "FLAVORS_TO_BUILD=$(make flavors.modified)" )
```

Comment:
Same remark about the complexity.
Why not doing "make FLAVORS_TO_BUILD=$(make flavors.modified)" ?